csrankings.csv

```diff
@@ -20700,3 +20700,20 @@ m. c. schraefel,University of Southampton,https://www.ecs.soton.ac.uk/people/sig
 Öznur Özkasap,Koç University,http://home.ku.edu.tr/~oozkasap,v2WiJeIAAAAJ
 Øystein Nytrø,NTNU,https://www.ntnu.edu/employees/nytroe,-Zavs80AAAAJ
 Ümit V. Çatalyürek,Georgia Institute of Technology,https://www.cc.gatech.edu/~umit,OLDMURQAAAAJ
+Joelle Pineau,McGill University,https://www.cs.mcgill.ca/~jpineau/,CEt6_mMAAAAJ

```

Comment:
Runs FAIR Montreal, therefore presumably not > 75% at McGill.

Reply:
She runs FAIR Montreal too, however she is a full-time faculty who supervises grad students at Mila and McGill. She has her own group of students from McGill, she has her office, and a name in the CS website. Not sure why she wouldn't qualify

Reply:
She is an associate professor as opposed to being adjunct therefore she is in a tenure track. Please let me know if you need additional info.

Reply:
Because you have to be 75% time to be included, and unless Prof. Pineau is spending just one day a week at FAIR, she doesn't qualify (like others at FAIR).

Reply:
I'm not sure about the exact percentages. The ranking and the research index doesn't reflect the true score if Joelle is excluded from the csrankings website. She is one of the very important faculties in the AI field at McGill CS dept.

Reply:
This is standard across CSrankings, and is the policy. Full-time faculty are included; non full-time faculty (< 75%) are not.

Reply:
But you don't know if she is <75%. Doina heads Deepmind, Montreal, Yoshua is a partner with a lot of startups in Mila. It doesn't mean they are <100% at McGill or UdeM.

Reply:
FYI https://www.cifar.ca/bio/joelle-pineau. She is still a full-time faculty at McGill.

Reply:
I fail to see how directing a lab is compatible with a one day a week commitment. Faculty routinely consult for 1 day a week (20%), and that's fine. The person you point out should probably also be removed.

Reply:
I think you are just being biased here. A lot of AI faculty (easily >20) that are already on the website head a particular industrial lab or a startup. If you want to be fair, you should probably remove the names of most faculties from top US universities like UC Berkeley, CMU, etc.

On a different note, I think csrankings are misleading many undergrads because you have many hidden rules (to list a faculty on your website) which is not displayed on the website. Also, you have included many faculties like Sergey Levine, etc who are also a part of industrial lab, but don't want to include faculties when it comes to McGill University.

Reply:
Also I fail to see why heading a lab implies they are <75% faculty at a University like you are mentioning. Afaik, this is not the case, at least at McGill University.

Reply:
You are also incorrectly concluding that a particular faculty is consulting one day a week (20%). Can you please provide additional details that led to the above conclusion

Reply:
I am saying that it is _fine_ for faculty to consult one day a week. More than that means they are not full-time. This is clearly stated on the FAQ. It's not a surprise that the database is not perfect. It has thousands of faculty from around the world, and it is literally a one-person effort. Anyway, the policy stands.

Reply:
I do understand your effort @emeryberger. I had a very high regard for the website you create. It is not easy for an international student to find a good research centric ranking website which can be trusted. That's why I regarded it very high.

Reply:
However, interactions with you over the pull request has given me a different perspective. I have a first hand experience collaborating with the faculties at McGill and I personally know who is full-time and who isn't. However, you seem to veto my first-person experience (supported by info from various websites) to take a call to remove very reputed faculties from the website. You are also incorrectly concluding that a faulty heading an industrial lab implies they are <75% at the University. I'm sorry to say that I'm disappointed by your faulty decisions.

Reply:
Full-time status is not actually something you can necessarily know (it's a financial question, at the end of the day). If the faculty members in question would like to comment on this thread to stipulate that they are in fact employed >75% at their institution, that would suffice (I'm sorry but it can't be second-hand, and websites are generally lagging indicators at best). I am enforcing a global policy here. If you can't understand that, that's unfortunate.

Reply:
I do understand the policy. I'm saying I would like to see the policy being enforced on many other Universities (not just McGill).

Reply:
That's fine: the way this works in general is that people either post PRs that remove faculty because of a move to industry (moving them into the `old/industry.csv`) or they contact me directly, if they'd prefer not to do this publicly, and provide corroborating documentation either way.

Reply:
And I can assure you, this has been done for faculty at a variety of universities (see `old/industry.csv` for some examples).

Reply:
The problem is you are adding or removing the faculty at your discretion. As pointed out by yourself, there is no way to find out if a particular faculty is >75% without looking at their salary/financial details. If this is the case, how to you make the call to approve a PR? How do you verify if it is correct and fits your policy?

Reply:
This is based on (numerous) past discussions with other faculty in similar arrangements.